### PR TITLE
Add row Actions to the Table

### PR DESCRIPTION
**Pull request by fhlavac**
https://issues.redhat.com/browse/HAC-1713

Added possibility to define RowActions

actions

@vidyanambiar 
@florkbr

### CODE REVIEW

File: packages/lib-utils/src/components/table/VirtualizedTableBody.tsx [hunk 1]
```diff
@@ -128,6 +131,11 @@ const VirtualizedTableBody = <D,>({
             />
           )}
           <RowMemo Row={Row} obj={rowArgs.obj} />
+          {rowActions ? (
```
Comment (vojtechszocs) [inline packages/lib-utils/src/components/table/VirtualizedTableBody.tsx:134]:
```suggestion
          {rowActions && (
            <Td isActionCell>
              <ActionsColumn items={rowActions} />
            </Td>
          )}
```

**Review by vojtechszocs — CHANGES_REQUESTED** [review]
Some minor comments, looks good overall.

**Review by vidyanambiar — COMMENTED** [review]
Thanks @fhlavac!
/lgtm

**Review by vojtechszocs — APPROVED** [review]

**Pull request merged**